src/lib/y2partitioner/widgets/pages/disk.rb

```diff
@@ -139,7 +140,18 @@ def label
         # @macro seeCustomWidget
         def contents
           # Page wants a WidgetTerm, not an AbstractWidget
-          @contents ||= VBox(DiskDeviceDescription.new(@disk))
+          @contents ||= VBox(description)
```

Comment:
Please, mention in the general description of `Pages::Disk` that it's used also for bcache devices.